src/components/App/App.js

```diff
@@ -7,10 +8,25 @@ import { Footer } from "../Footer/Footer";
 import SearchPage from "../SearchPage/SearchPage";
 import StoresPage from "../StoresPage/StoresPage";
 import CartPage from "../CartPage/CartPage";
+import { GET_USER } from "../../hooks/getQueries";
+import { useLazyQuery, useMutation } from "@apollo/client";
 
 const App = () => {
+  const [user, setUser] = useState(0);
+  const [getUser, { loading, error }] = useLazyQuery(GET_USER, {
```

Comment:
If you wanted to have the App not load until the user has been set (which is ideal), I would use something like if(loading) {
return null, else - then load the app, about line 13.